internal/shared/types/app_manifest.go

```diff
@@ -76,6 +76,12 @@ type AppFeatures struct {
 	ManifestShortcutsItems     []ManifestShortcutsItem     `json:"shortcuts,omitempty" yaml:"shortcuts,flow,omitempty"`
 	ManifestSlashCommandsItems []ManifestSlashCommandsItem `json:"slash_commands,omitempty" yaml:"slash_commands,flow,omitempty"`
 	Search                     *Search                     `json:"search,omitempty" yaml:"search,flow,omitempty"`
+	RichPreviews               *RichPreviews               `json:"rich_previews,omitempty" yaml:"rich_previews,flow,omitempty"`
+}
+
+type RichPreviews struct {
+	EntityTypes []string `json:"entity_types,omitempty" yaml:"entity_types,flow,omitempty"`
+	IsActive    bool     `json:"is_active,omitempty" yaml:"is_active,flow,omitempty"`
```

Comment:
I was on the fence about including this property but after some discussion, we might want to support the dev setting this rather than needing to remove the entire rich_previews setting to disable (so I added it back)